lib/ansible/modules/packaging/os/yum.py

```diff
@@ -43,6 +43,9 @@
   state:
     description:
       - Whether to install (C(present) or C(installed), C(latest)), or remove (C(absent) or C(removed)) a package.
```

Comment:
How about just this fix?
```
- Whether to install (C(present) or C(installed)), update (C(latest)) or remove (C(absent) or C(removed)) a package.
```
It seems clear enough for me. However, just a suggestion, I am fine with merging this PR.